services/CertificateGeneration/CertificateGeneration/CertificateGeneration.csproj

```diff
@@ -156,6 +156,11 @@
         </WebProjectProperties>
       </FlavorProperties>
     </VisualStudio>
+    <MonoDevelop>
+      <Properties>
+        <XspParameters Port="8080" Address="127.0.0.1" SslMode="None" SslProtocol="Default" KeyType="None" CertFile="" KeyFile="" PasswordOptions="None" Password="" Verbose="True" />

```

Comment:
Is it ok to use 8080?

Reply:
Looks like this was auto-added by vs community. Removed.